pkg/scheduler/apis/config/scheme/scheme_test.go

```diff
@@ -178,11 +179,25 @@ profiles:
       apiVersion: kubescheduler.config.k8s.io/v1beta1
       kind: InterPodAffinityArgs
 `),
-			wantErr: "decoding .profiles[0].pluginConfig[0]: args for plugin NodeLabel were not of type NodeLabelArgs.kubescheduler.config.k8s.io, got InterPodAffinityArgs.kubescheduler.config.k8s.io",
+			wantErr: `decoding .profiles[0].pluginConfig[0]: args for plugin NodeLabel were not of type NodeLabelArgs.kubescheduler.config.k8s.io, got InterPodAffinityArgs.kubescheduler.config.k8s.io`,
```

Comment:
any particular reason for this change?

Reply:
I usually prefer to use backticks in exact string matching to avoid spending too much time trying to debug whitespaces. I changed this one as well to have it consistent in this file.

Reply:
sg